assets/app/scripts/filters/util.js

```diff
@@ -211,6 +211,11 @@ angular.module('openshiftConsole')
       return link;
     };
   })
+  .filter('eventIcon', function() {
+    return function(type) {
+      return (type === "Normal") ? "pficon-info" : "pficon-warning-triangle-o";
```

Comment:
Would rather have a switch here with a default case that returns `""`. If there are ever new event types, they'll show a warning icon.